core/app/components/FormBuilder/FormElement.tsx

```diff
@@ -92,9 +98,10 @@ export const FormElement = ({ element, index, isEditing, isDisabled }: FormEleme
 						{restoreRemoveButton}
 						<Button
 							type="button"
+							aria-label="Edit field"
 							disabled={isDisabled || element.deleted}
 							variant="ghost"
-							className="invisible p-2 group-hover:visible"
+							className="invisible p-2 group-hover:visible group-focus:visible"
```

Comment:
Now that keyboard interactions work, this makes sure the buttons are visible if you tab to a form element. I still haven't quite figured out how to make them appear in the tab order though

Reply:
yeah the tabindex not changing after reordering is why I kind of gave up on this for now. thanks for figuring out the other part though, I'll apply those changes!